### ISSUE TITLE
No suggestion

### ISSUE THREAD

**Issue by starshipcoder** [chinalwb/Android-Rich-text-Editor#127]
To be able to use getSelectionStart(), the suggestions was deactivated

If would be great to found another way to span only the wanted piece of text that allow the suggestions to work

**Comment by chinalwb** [comment]
Yes @starshipcoder !  By disabling the suggestions I meant to avoid the side effect between the suggestion styles and the rich text styles. I agree if we can have the suggestion feature works well with the existing styles that would be great!

BTW, thanks for all of the fantastic PRs!!
